package io.sharing.server.causer.support.exception

class TooManyAuthCodeRetryException : Exception()
